docs/databases/connections/databricks.md

```diff
@@ -39,20 +39,25 @@ See [Personal Access Token (PAT)](https://docs.databricks.com/en/dev-tools/auth/
 
 See [Authenticate access with a service principal using OAuth](https://docs.databricks.com/en/dev-tools/auth/oauth-m2m.html).
 
-### Catalog
-For now, you can only select one catalog. Metabase doesn't support multi-catalog connections. If you want to use more than one catalog in Metabase, you can set up multiple connections, each selecting a different catalog.
+### Enable multiple catalogs
+
+Toggle on to sync multiple catalogs. If you enable this, you'll be able to [filter catalogs](#catalogs-and-schemas).
```

Comment:
This makes it seem
```suggestion
Toggle on to sync multiple catalogs. If you enable this, you'll be able to [specify which catalogs to sync](#catalogs-and-schemas).
```

otherwise it seems like _restricting_ the catalogs is the selling point, but it's the other way around